apps/avatax/package.json

```
@@ -5,6 +5,7 @@
     "build": "pnpm generate && next build",
     "dev": "pnpm generate && NODE_OPTIONS='--inspect' next dev",
     "e2e": "vitest --project e2e --watch=false",
+    "e2e:watch": "vitest --project e2e --watch",
```

```suggestion
    "e2e:watch": "vitest --project e2e",
```

I think by default vitest will run in watch mode